src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -136,9 +136,6 @@ $pf-global--spacer--form-element: pf-size-prem(6px) !default;
 $pf-global--gutter: $pf-global--spacer--lg !default;
 $pf-global--gutter--md: $pf-global--spacer--md !default;
 
-// Ratio
-$pf-global--golden-ratio: 1.681 !default;
```

Comment:
Removing this is a breaking change, too. Let's add it back and leave a comment in the code to remove it later or create an issue or however you'd prefer deal with it.

Reply:
Okay added them back in captured it in an issue here: #2054